tilequeue/command.py

```diff
@@ -983,27 +1021,42 @@ def tilequeue_prune_tiles_of_interest(cfg, peripherals):
     prune_cfg = cfg.yml.get('toi-prune', {})
 
     redshift_cfg = prune_cfg.get('redshift', {})
-    redshift_uri = redshift_cfg.get('database-uri')
-    assert redshift_uri, ("A redshift connection URI must "
+    db_conn_info = redshift_cfg.get('database-uri') or cfg.postgresql_conn_info
+    assert db_conn_info, ("A redshift connection URI or postgres configuration must "
                           "be present in the config yaml")
+    
+    is_postgres_conn_info = isinstance(db_conn_info, dict)
+    if is_postgres_conn_info:
```

Comment:
I'd like to see this block go away when the `redshift` section in config simply has a database URI.